packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -750,6 +751,9 @@ function emitBasicOperation(
   });
 
   for (const param of httpOperation.parameters.parameters) {
+    if (isIgnoredHeaderParam(param)) {
```

Comment:
Three headers need to be ignored see https://github.com/Azure/typespec-azure/issues/3429